packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/ExpertOptions/ExpertOptionsContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react'
+import { MobXProviderContext, observer } from 'mobx-react'
+import ExpertOptions from './ExpertOptions'
+
+function useStores() {
+  const stores = React.useContext(MobXProviderContext)
+
+  const {
+    authClient
+  } = stores.classifierStore
+  return {
+    authClient
+  }
+}
+
+function ExpertOptionsContainer (props) {
+  const { authClient } = useStores()
+  const [isAdmin, setIsAdmin] = useState(false)
+
+  useEffect(() => {
+    authClient.checkCurrent().then((user) => {
+      if (user) {
+        setIsAdmin(user.admin)
+      } else {
+        setIsAdmin(false)
+      }
+    }).catch((error) => { console.error(error) })
+
+    return setIsAdmin(false)
```

Comment:
Sorry, missed this on my first pass. I think useEffect is supposed to return a callback function.
```suggestion
    return () => setIsAdmin(false)
```

Reply:
My suggestion could also be a named function, to make debugging easier.
```js
return function onUnmount() {
  setIsAdmin(false)
}
```